publications.html

```diff
@@ -142,7 +155,7 @@
 
     <section>
       <div class="article container">
-        <a class="title" href="https://epiclab.github.io/ICSE19-LatentPatternsInActivities/">Latent Patterns in Activities: A Field Study of How Developers Manage Context
+        <a class="title" href="https://dl.acm.org/doi/10.1109/ICSE.2019.00051">Latent Patterns in Activities: A Field Study of How Developers Manage Context
```

Comment:
Several publications have links to artifacts (e.g. survey formats, anonymized datasets, etc.), which are included as URLs within the actual finalized publications and are distributed as anonymized links for reviewers. The [https://epiclab.github.io/ICSE19-LatentPatternsInActivities/](https://epiclab.github.io/ICSE19-LatentPatternsInActivities/) page is one of them, and it should not be removed. Direct links to the paper can be updated on those individual publication pages if needed.